item-spec/item-spec.md

```diff
@@ -73,11 +73,10 @@ In general, STAC versions can be mixed, but please keep the [recommended best pr
 
 #### stac_extensions
 
-A list of extensions the Item implements.  
-This list must only contain extensions that extend the Item itself, see the the 'Scope' column in the list of 
-extensions. The list contains URLs to the JSON Schema files it can be validated against.
-If an extension such as the `tiled-assets` extension has influence on multiple parts of the whole catalog 
-structure, it must be listed in all affected parts (e.g. Catalog, Collection and Item for the `tiled-assets` extension).
+A list of extensions the Item implements.
```

Comment:
thanks for rewriting this! I updated this yesterday and only removed the outdated text about shortnames. This sounds much better.